### ISSUE TITLE
Feature: scatterplot.select() with possibility to suppress triggering a select event

### ISSUE THREAD

**Issue by plankter** [flekschas/regl-scatterplot#18]
Is it possible to have an option in `scatterplot.select()` method to suppress triggering a select event (probably a new method)? This is useful in such scenarios like cross-filtering between two plots: one selects points on one plot and these points should be highlighted in another plots as well. At the moment this action calls select handler, which creates an infinite loop.

**Comment by flekschas** [comment]
Yeah, I came across that too and wonder what the best solution would be.

One idea is to add an option-object with a property to suppress the subsequent event. E.g.:

```javascript
scatterplot.select(points, { silent: true });
```

I wonder what to call the property. I can think of `silent: true` or `notify: false`. Any other ideas?

I also wonder how other JS libraries handle this case.

**Comment by plankter** [comment]
A proper naming is always an issue for me too. I was thinking about something like a default method signature

```js
scatterplot.select(points, suppress = false);
```
 when user can change default behavior by setting `suppress = true`. But any implementation will do, of course.

**Comment by flekschas** [comment]
Addressed in 524716e

You can now do `scatterplot.select(points, { preventEvent: true });` to prevent the event from being published.

Will be part of the upcoming `v0.10`
